Providers/AppsFlyerProvider.m

```diff
@@ -33,7 +33,11 @@ - (void)identifyUserWithID:(NSString *)userID andEmailAddress:(NSString *)email
 }
 
 - (void)event:(NSString *)event withProperties:(NSDictionary *)properties {
-    [[AppsFlyerTracker sharedTracker] trackEvent:event withValue:@""];
+    if (properties != nil) {
+        [[AppsFlyerTracker sharedTracker] trackEvent:event withValues:properties];
```

Comment:
why don't we always call the call `trackEvent: withValues:` ? (even when properties is nil)